From eb92a8ee969ac1b2128d8091ae9e1c4829e7dbca Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Wed, 22 Jul 2015 16:01:37 -0500
Subject: [PATCH] ARM: dts: dra74x: Remove non-common cpu0 operating-points

Remove all OPPs that are not common to all dra74x variants.

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
---
 arch/arm/boot/dts/dra74x.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/dra74x.dtsi b/arch/arm/boot/dts/dra74x.dtsi
index cc560a7..168b86e 100644
--- a/arch/arm/boot/dts/dra74x.dtsi
+++ b/arch/arm/boot/dts/dra74x.dtsi
@@ -24,7 +24,6 @@
 			operating-points = <
 				/* kHz    uV */
 				1000000	1060000
-				1176000	1160000
 				>;
 
 			clocks = <&dpll_mpu_ck>;
-- 
2.1.4

